src/GraphQL.Tests/Bugs/Bug1205VeryLongInt.cs

```diff
@@ -32,9 +34,9 @@ public void Very_Long_Number_In_Input_Should_Return_Error_For_Int()
             {
                 Errors = new ExecutionErrors
                 {
-                    new ExecutionError("Argument \"in\" has invalid value 636474637870330463.\nExpected type \"Int\", found 636474637870330463.")
+                    new ValidationError(null, ArgumentsOfCorrectTypeError.NUMBER, "Argument \"in\" has invalid value 636474637870330463.\nExpected type \"Int\", found 636474637870330463.")
```

Comment:
Why don't you use `new ArgumentsOfCorrectTypeError()`?

Reply:
I'd have to make additional constructors that don't require `ValidationContext`, etc

Reply:
OK